frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -1367,7 +1367,14 @@ class DataApi {
       V3.add([curX, curY, curZ], [halfViewportExtentX, halfViewportExtentY, 1]),
       viewport,
     );
-    const resolutionIndex = getRequestLogZoomStep(state);
+
+    // Find a viable resolution to compute the histogram on
+    const layer = getLayerByName(state.dataset, layerName);
+    const resolutionInfo = getResolutionInfo(layer.resolutions);
+    // Ideally, we want to avoid resolutions 1 and 2 to keep
+    // the amount of data that has to be loaded small
+    const maybeResolutionIndex = Math.max(2, getRequestLogZoomStep(state));
```

Comment:
How about changing this to:

```suggestion
    const maybeResolutionIndex = Math.max(2, getRequestLogZoomStep(state) + 1);
```

That way, the histogram clipping will be way faster (factor 8 in isotropic cases), since it would load the fallback data in most cases. Otherwise, all best-quality buckets for the current mag (if it's greater than mag 2) would need to be awaited which can take some seconds.

Reply:
I see :) Thanks!